Userland/Libraries/LibWeb/Streams/CountQueuingStrategy.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, Shannon Booth <shannon.ml.booth@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Forward.h>
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Forward.h>
+#include <LibWeb/Streams/QueuingStrategyInit.h>
+
+namespace Web::Streams {
+
+// https://streams.spec.whatwg.org/#countqueuingstrategy
+class CountQueuingStrategy final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(CountQueuingStrategy, Bindings::PlatformObject);
+
+public:
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<CountQueuingStrategy>> construct_impl(JS::Realm&, QueuingStrategyInit const&);
+
+    virtual ~CountQueuingStrategy() override;
+
+    // The highWaterMark getter steps are:
```

Comment:
Wire up the spec link (https://streams.spec.whatwg.org/#cqs-high-water-mark) here, and move this comment inside the getter.

Reply:
Ah, thanks, closer to how other places does it. Applied this change, and all others suggested. Thanks for the quick review! I'm still trying to figure out the size steps, a little confused by the spec and how to implement that right now :D